pixelwall/pixelwall.h

```diff
@@ -0,0 +1,77 @@
+/*
+DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE
+                    Version 2, December 2004
+
+Copyright (C) 2004 Sam Hocevar <sam@hocevar.net>
+
+Everyone is permitted to copy and distribute verbatim or modified
+copies of this license document, and changing it is allowed as long
+as the name is changed.
+
+DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE
+TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION AND MODIFICATION
+
+0. You just DO WHAT THE FUCK YOU WANT TO.
+*/
+
+#pragma once
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>  
+#include <unistd.h>
+
```

Comment:
On linux i get an error:
design_random_pixels.c:18:1: note: ‘uintptr_t’ is defined in header ‘<stdint.h>’; did you forget to ‘#include <stdint.h>’?
Add here: #include <stdint.h>